mdsobjects/cpp/datastreams.cpp

```diff
@@ -431,7 +431,7 @@ EXPORT void EventStream::send(int shot, const char *name, int numSamples, uint64
 
 
 
-EXPORT void EventStream::run()
```

Comment:
I think you should put the export back for non WINDOWS_H otherwise you might not be able to use the stream